pkg/ebpf/tracer.go

```diff
@@ -127,6 +137,7 @@ func NewFlowFetcher(cfg *FlowFetcherConfig) (*FlowFetcher, error) {
 		var newObjects NewBpfObjects
 		// remove tcpdrop hook from the spec
 		delete(spec.Programs, tcpDropHook)
+		delete(spec.Programs, )
```

Comment:
there are some compilation errors here and above, I guess it wasn't intended to be commited?